common/persistence/elasticsearch/esVisibilityStore.go

```diff
@@ -817,6 +817,22 @@ func processAllValuesForKey(dsl *fastjson.Value, keyFilter func(k string) bool,
 	return nil
 }
 
+func combinedKeyFilter(key string) bool {
```

Comment:
Nit: isCombinedKeyFilter / isCompoundKeyFilter

Reply:
Makes sense. Updated to `isCombinedKey`, `isTimeKey` and `isCloseStatusKey`.